scripts/transparencia_dados_abertos_brasil.json

```diff
@@ -0,0 +1,73 @@
+{
+    "citation": "NA",
+    "description": "Tabelas com o levantamento de portais estaduais e municipais de transparência e dados abertos, feito originalmente por Rodrigo Klein em sua tese de doutorado na PUC/RS, em 2017",
+    "encoding": "utf-8",
+    "homepage": "https://github.com/augusto-herrmann/transparencia-dados-abertos-brasil",
+    "keywords": ["Brazil"],
+    "licenses": [{"name": "MIT"}],
+    "name": "transparencia-dados-abertos-brasil",
+    "resources": [
+        {
+            "dialect": {},
+            "name": "portais-estaduais",
+            "schema": {
+                "fields": [
+                    {
+                        "name": "uf",
+                        "type": "char"
+                    },
+                    {
+                        "name": "uf_nome",
+                        "type": "char"
+                    },
+                    {
+                        "name": "url",
+                        "type": "char"
+                    },
+                    {
+                        "name": "tipo",
+                        "type": "char"
+                    }
+                ]
+            },
+            "url": "https://raw.githubusercontent.com/augusto-herrmann/transparencia-dados-abertos-brasil/master/data/portais-estaduais.csv"
+        },
+        {
+            "dialect": {},
+            "name": "portais-municipais",
+            "schema": {
+                "fields": [
+                    {
+                        "name": "uf",
+                        "type": "char"
+                    },
+                    {
+                        "name": "município",
+                        "type": "char"
+                    },
+                    {
+                        "name": "população",
+                        "type": "int"
+                    },
+                    {
+                        "name": "url",
+                        "type": "char"
+                    },
+                    {
+                        "name": "tipo",
+                        "type": "char"
+                    },
+                    {
+                        "name": "observação",
+                        "type": "char"
+                    }
+                ]
+            },
+            "url": "https://raw.githubusercontent.com/augusto-herrmann/transparencia-dados-abertos-brasil/master/data/portais-municipais.csv"
+        }
+    ],
+    "retriever": ["True"],
+    "retriever_minimum_version": "2.1.0",
```

Comment:
Since this involves the new encoding work does the minimum version need to be "2.3.2" or "2.4.0" depending on what version bump you're using?